### DISCUSSION TITLE
LAB 03.02 Task 2 Error

### DISCUSSION THREAD

**Discussion by JonathanCh022** [rramosp/ai4eng.v1#59] [03 - GENERACION DE MODELOS]
Estoy en la tarea 2 del laboratorio 3.02 donde al crear mi función retorno un array de numpy como sugiere el ejercicio, pero al hacerlo el calificador no me arroja nota, me arroja el siguiente error. 
![image](https://user-images.githubusercontent.com/17456510/127401101-5eab00b7-10c7-493c-be9f-d6b7c08ee3d7.png)

cuando retorno otro tipo de dato como dataframe o list me califica con 0, pero al retornar numpy me sale ese error, quisiera saber que puedo hacer ahi.  pongo una copia de mi colab. muchas gracias por su atencion

https://colab.research.google.com/drive/10z4cWMECBDexjDeqyM8wbni4qGk832wv?usp=sharing

**Comment by Felipe-RA** [comment]
Saludos Jonathan

Te comento que no puedo acceder al colab por los permisos de acceso que tiene

Sin embargo, puedo observar viendo el traceback que el problema radica en la inexistencia del nombre "`np`". Esto usualmente ocurre al no importar un modulo o usar un nombre distinto al que tiene asignado en el `namespace`.

Verifica que si estes importando `numpy `y que si estes asignandole el nombre `np`.

Por ejemplo:

`import numpy as np`

**Reply by JonathanCh022** [reply]
creo que ya cambie la configuración, no me di cuenta cuando copie el link, pido excusas por eso. 
respecto a lo de importar numpy as np, lo ejecuto en la tercera celda del notebook y por curiosidad lo coloque en la misma celda de la función del ejercicio pero el error aun persiste.

**Comment by JonathanCh022** [comment]
![image](https://user-images.githubusercontent.com/17456510/127411872-9adf6ffc-6a55-4ea8-8699-9d76e137c769.png)

Una captura del codigo y un nuevo link sin restricciones del colab. 
quedo atento, muchas gracias por el curso, esta muy bueno.

https://colab.research.google.com/drive/10z4cWMECBDexjDeqyM8wbni4qGk832wv?usp=sharing

**Reply by JonathanCh022** [reply]
Listo ya lo solucione, el problema estaba en que no colocaba el import dentro de la función.  Muchas gracias 
![image](https://user-images.githubusercontent.com/17456510/127509519-009dc1fd-7897-4b89-984d-f0f2fa0fe590.png)
